client/bindings/nodejs/lib/Client.ts

```diff
@@ -689,6 +689,20 @@ export class Client {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Returns the metadata of the included block of the transaction.
+     */
+    async getIncludedBlockMetadata(transactionId: string): Promise<IBlock> {
```

Comment:
If you run `npm run docs-wiki-build` it will build the API ref docs for it
For the python binding `pydoc-markdown -p iota_client > ../../../documentation/docs/libraries/python/api_reference.md`

Reply:
Thanks